addons/Text_Coloring/src/presenter.js

```diff
@@ -1336,6 +1354,10 @@ function AddonText_Coloring_create() {
     };
 
     presenter.setShowErrorsMode = function () {
+        if(presenter.configuration.isNotActivity) {
+            return 0;
```

Comment:
Niepotrzebne zwrócenie 0 - wprowadza lekkie zamieszanie. 
Jednak nie powinno to niczego psuć.
Jeżeli ticket wróci to proszę o poprawkę.